app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if len(command.split(" ")) == 3:
+        command, file, new_file = command.split(" ")
```

Comment:
There is no need to perform a `split` operation twice, make it once and write an obtained list to a variable that you will use for length checking and unpacking.